src/components/AmountFilter.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Button, Popover, InputLabel, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import RemoveIcon from '@material-ui/icons/Remove';
+import AmountTextField from './AmountTextField';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: `${theme.spacing(2)}px ${theme.spacing(1)}px 0`,
+  },
+  filterStyle: { margin: `${theme.spacing(1)}px 0` },
+  labelStyle: { paddingLeft: theme.spacing(1) },
+  dashStyle: {
+    paddingTop: theme.spacing(2),
+    display: 'flex',
+    alignItems: 'center',
+  },
+}));
+
+export default function AmountFilter() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [values, setValues] = useState({});
+
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+  const handleChange = (index, val) =>
+    setValues({ ...values, [index]: parseInt(val, 10) || '' });
+
+  return (
+    <>
+      <Button
+        variant="contained"
```

Comment:
I think an [outlined button](https://material-ui.com/components/buttons/#outlined-buttons) would actually look more like the Figma design